src/components/modal/ModalAddPatient.tsx

```diff
@@ -57,45 +57,6 @@ const ModalAddPatient: React.FC<ModalAddPatientProps> = ({
             />
           </Form.Group>
 
-          <Form.Group as={Col} className='mt-2'>
-            <Form.Label><FontAwesomeIcon icon={['fas', 'address-card']} /> {t('form.emailAddress')}</Form.Label>
-            <Form.Control
-              name='address'
-              type='text'
-              value={patientState.street}
-              onChange={handleChangeNewPatient}
-            />
-          </Form.Group>
-
-          <Form.Group as={Col} className='mt-2'>
-            <Form.Label><FontAwesomeIcon icon={['fas', 'mobile']} /> {t('form.phone')}</Form.Label>
-            <Form.Control
-              name='phone'
-              type='text'
-              value={patientState.phone}
-              onChange={handleChangeNewPatient}
-            />
-          </Form.Group>
-
-          <Form.Group as={Col} className='mt-2'>
-            <Form.Label><FontAwesomeIcon icon={['fas', 'city']} /> {t('form.city')}</Form.Label>
-            <Form.Control
-              name='city'
-              type='text'
-              value={patientState.city}
-              onChange={handleChangeNewPatient}
-            />
-          </Form.Group>
-
-          <Form.Group as={Col} className='mt-2'>
-            <Form.Label><FontAwesomeIcon icon={['fas', 'city']} />  {t('form.zipPostal')}</Form.Label>
-            <Form.Control
-              name='zip_code'
-              type='text'
-              value={patientState.zip_code}
-              onChange={handleChangeNewPatient}
-            />
-          </Form.Group>
```

Comment:
Why not implementing what we suggested using collapse?
https://react-bootstrap-v4.netlify.app/components/accordion/

Reply:
I will create a ticket for it because it is not urgently

Reply:
https://github.com/mynotif/mynotif-frontend/issues/235